src/sushipy/main.py

```diff
@@ -44,7 +44,6 @@ def _extend_check(self) -> None:
                 obj.install()
 
     def __init__(self) -> None:
-        chdir(getcwd())  # todo: do we really need that?

```

Comment:
Yeah, it isn't needed anymore. I added this because I had some problems with my environment.